### ISSUE TITLE
Input pipe is not supported

### ISSUE THREAD

**Issue by endmarsfr** [fraunhoferhhi/vvenc#5]
Hi,

I tried with no joy  following command 
`ffmpeg.exe -i "input.avs" -hide_banner -nostdin -f yuv4mpegpipe -pix_fmt yuv420p - | vvencapp.exe --input - --threads 6 --preset faster  --format yuv420 --qp 32 --framerate 25.0 --size 1920x1080 --output "output.266"`
but unfortunatelly, pipe is not yet supported, it is absolutely necessary because yuv files are so heavy !

could you please plan that in next version ? 

regards

**Comment by 1480c1** [comment]
if this is on bash, you can use
```bash
vvencapp.exe --input <(ffmpeg.exe -i "input.avs" -hide_banner -nostdin -f yuv4mpegpipe -pix_fmt yuv420p -) --threads 6 --preset faster --format yuv420 --qp 32 --framerate 25.0 --size 1920x1080 --output "output.266"
```
until official support is added

**Comment by endmarsfr** [comment]
Thanks a lot for the workaround,it worked using bash if I prefix with $
`vvencapp.exe --input < $(ffmpeg.exe ....)  ....`

**Comment by 1480c1** [comment]
note the lack of `$`, this is not a regular replacement subshell

**Comment by endmarsfr** [comment]
Thank you, 
you're, right, I tried with `--input <(ffmpeg.exe ...)` with no space between < and (   👍 
it worked better, but  I used "msys2" bash and I get now `[error]: failed to open input file /proc/1805/fd/63`

**Comment by master-of-zen** [comment]
That would be amazing

**Comment by master-of-zen** [comment]
Every day without piping for VVenC is suffering :(

**Comment by adamjw24** [comment]
For now this is blocked by the lack of "sliding window" mode for the RC, which is not planned for the next release yet.

Possibly we will investigate a mode like that earlier for single pass and fixed QP encoding earlier, though. Also seems like a good issue to contribute.

**Comment by adamjw24** [comment]
implemented in #45

**Comment by AtenaNg** [comment]
Hi, I use this command on cmd and still get the error. Any suggestion on this?

`ffmpeg.exe -i "input.mp4" -nostdin -f yuv4mpegpipe -pix_fmt yuv420p - | vvencFFapp.exe --input - --threads 6 --preset faster --format yuv420 --qp 32 --framerate 25.0 --size 1920x1080 --output "output.266"`

Also this command on bat file returned the same error 

`vvencFFapp.exe --input <(ffmpeg -i "input.mp4" -hide_banner -nostdin -f rawvideo -) --threads 6 --preset faster --format yuv420 --qp 32 --framerate 25.0 --size 1920x1080 --output output.266
`
![image](https://user-images.githubusercontent.com/88569834/129375408-72d4e605-e983-4e6b-9b82-680dc46fd064.png)
